tests/integration_tests/test_tracegraph_elbo.py

```diff
@@ -568,8 +568,8 @@ def guide():
                                             infer=dict(baseline=dict(baseline_value=baseline_value)))
                         assert z_i_k.shape == (4 - i,)
 
-        def per_param_callable(module_name, param_name, tags):
-            if 'baseline' in tags:
+        def per_param_callable(module_name, param_name):
+            if 'baseline' in param_name or 'baseline' in module_name:
```

Comment:
:+1: I think string matching is a more maintainable mechanism than a tags dict + code to access it. Especially as we refactor towards local param stores.